mitxgraders/plugins/defaults_sample.py

```diff
@@ -9,18 +9,18 @@
 This plug-in works by registering a dictionary with the appropriate grading class (note,
 not class instance).
 """
-from __future__ import print_function, division, absolute_import, unicode_literals
-
-# Here are all of the graders that you can specify defaults for
-from mitxgraders.stringgrader import StringGrader
-from mitxgraders.baseclasses import AbstractGrader, ItemGrader
-from mitxgraders.listgrader import ListGrader, SingleListGrader
-from mitxgraders.formulagrader.integralgrader import IntegralGrader
-from mitxgraders.formulagrader.formulagrader import FormulaGrader, NumericalGrader
-from mitxgraders.formulagrader.matrixgrader import MatrixGrader
-
-# These will be needed to set attempt-based credit course-wide
-from mitxgraders.attemptcredit import LinearCredit, GeometricCredit, ReciprocalCredit
+# 
+
+# # Here are all of the graders that you can specify defaults for
+# from mitxgraders.stringgrader import StringGrader
+# from mitxgraders.baseclasses import AbstractGrader, ItemGrader
+# from mitxgraders.listgrader import ListGrader, SingleListGrader
+# from mitxgraders.formulagrader.integralgrader import IntegralGrader
+# from mitxgraders.formulagrader.formulagrader import FormulaGrader, NumericalGrader
+# from mitxgraders.formulagrader.matrixgrader import MatrixGrader
+
+# # These will be needed to set attempt-based credit course-wide
+# from mitxgraders.attemptcredit import LinearCredit, GeometricCredit, ReciprocalCredit
```

Comment:
Why are we commenting on this?

Reply:
I see. Wouldn't it be better to leave all this in the docstring?

Reply:
I think letting that file without changes is better because I commented the imports to avoid unused imports. Still, for the instructions in the docstring, the idea is to duplicate that file and use it to override things, and if I comment on the imports, we need to change the instructions, and the person should uncomment the things by themselves.

I'll revert this change.